eng/pipelines/libraries/build-job.yml

```diff
@@ -5,6 +5,7 @@ parameters:
   archType: ''
   framework: netcoreapp
   isOfficialBuild: false
+  liveCoreClrBuildConfig: ''
```

Comment:
Does `liveCoreClrBuildConfig` represent the version of CoreCLR we should be building and using or simply using? Or stated differently does this control whether we build CoreCLR here or merely which already built CoreCLR we should be consuming?

Reply:
The intention of this is to tell the `Libraries` build which CoreCLR build it should depend on. This way we can build `CoreCLR` for Release and then Libraries for Debug and depend on the CoreCLR release build, that way configurations are decoupled.

Reply:
Gotcha. @trylek consider adding a comment clarifying this.